_events/raffle.html

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Raffle
+permalink: /events/raffle
+---
+
+<script src="/assets/js/raffle.js"></script>
+<noscript><p>JavaScript is required to use the raffle machine.</p></noscript>
+<div id="raffle"></div>
```

Comment:
Can you please add a loading message here before the page initially loads and then hide it when all of the points data has been loaded? See the points page for an example on how to do that.

Reply:
Fixed in commit e21492a50546a29bdf814652e89762ccc1ffb69d.